src/ConfigChanges/ConfigFile.js

```diff
@@ -188,20 +188,16 @@ function resolveConfigFilePath (project_dir, platform, file) {
     // only if none of the options above are satisfied does this get called
     // TODO: Move this out of cordova-common and into the platforms somehow
     if (platform === 'android' && !fs.existsSync(filepath)) {
-        var config_file;
         if (file === 'AndroidManifest.xml') {
             filepath = path.join(project_dir, 'app', 'src', 'main', 'AndroidManifest.xml');
         } else if (file.endsWith('config.xml')) {
             filepath = path.join(project_dir, 'app', 'src', 'main', 'res', 'xml', 'config.xml');
         } else if (file.endsWith('strings.xml')) {
             // Plugins really shouldn't mess with strings.xml, since it's able to be localized
             filepath = path.join(project_dir, 'app', 'src', 'main', 'res', 'values', 'strings.xml');
-        } else if (file.includes(path.join('res', 'values'))) {
-            config_file = path.basename(file);
-            filepath = path.join(project_dir, 'app', 'src', 'main', 'res', 'values', config_file);
```

Comment:
I definitely don't think it's safe to lose these changes

Reply:
This `else if` was added back.